test/models/game_test.rb

```diff
@@ -13,4 +13,15 @@ class GameTest < ActiveSupport::TestCase
     assert_equal 2, game.kings.count, 'populate_board should create 2 Kings'
     assert_equal 2, game.queens.count, 'populate_board should create 2 Queens'
   end
+
+  test 'game is full' do
+    game = create(:game, white_user_id: 1, black_user_id: 2)
```

Comment:
We've got mixed hash syntax in this file. I'm not sure if we decided on hash rockets or the newer style. We should probably make sure it's consistent though.